cmd/paasta/zipkin_monk.go

```diff
@@ -15,7 +17,12 @@ const zipkinReporter = "monk"
 
 func initZipkin(zipkinURL string) (reporter.Reporter, *zipkin.Tracer, error) {
 	if zipkinURL == "" {
-		zipkinURL = "monk://169.254.255.254:1473/zipkin"
+		runtimeenv, err := ioutil.ReadFile("/nail/etc/runtimeenv")
```

Comment:
cna you put the monk endpoint & stream name in system paasta config instead so that we don't have to leak yelpy stuff like this into the code?